src/lib/vault/getFormSubmission.ts

```diff
@@ -0,0 +1,40 @@
+import { GetCommand } from "@aws-sdk/lib-dynamodb";
+import { dynamodbClient } from "@lib/awsServicesConnector";
+
+interface FormSubmission {
+  content: string;
+}
+
+export async function getFormSubmission(
+  formId: string,
+  submissionName: string,
+): Promise<FormSubmission | undefined> {
+  try {
+    const response = await dynamodbClient.send(
+      new GetCommand({
+        // biome-ignore lint/style/useNamingConvention: <explanation>
+        TableName: "Vault",
+        // biome-ignore lint/style/useNamingConvention: <explanation>
+        Key: { FormID: formId, NAME_OR_CONF: `NAME#${submissionName}` },
+        // biome-ignore lint/style/useNamingConvention: <explanation>
+        ProjectionExpression: "FormSubmission",
+      }),
+    );
+
+    if (response.Item === undefined) {
+      return undefined;
+    }
+
+    return {
+      content: response.Item.FormSubmission,
+    };
+  } catch (error) {
+    console.error(
+      `[dynamodb] Failed to retrieve form submission. FormId: ${formId} / SubmissionName: ${submissionName}. Reason: ${JSON.stringify(
```

Comment:
We will probably be looking at integrating Pino or something similar later on to replace our existing console logs.

In the GC Forms web app we are not always able to know what happened in the code because our logs are sometimes missing some context (layer, inputs, outputs, ect..). 

Here is an idea I would like to propose for a future log function:

````
log({
  level: "error",
  message: "Failed to retrieve form submission",
  cause: "Throughput exceeds the current capacity of your table or index",
  context: {
    layer: "dynamodb",
    inputs: {
      variable1: 2,
      variable2: "abc123",
    },
  },
});
````

Reply:
I've recommended a log schema here https://docs.google.com/document/d/1gfnmZ3POdwd2nsIi6w6zgAgvyNnReK76vh2RHdYy35g/edit#heading=h.kr0jyzxcrgwx curious what make sense in practice

Reply:
@wmoussa-gc Oh yeah right! I forgot about it 😞 We definitely have things in common from what I can see.

@bryan-robitaille Fully agree with what you said. There are some things we won't be able to include in our logs but for what is possible then I think we should do it. I believe it will help us a lot during our investigations.